client/src/components/About.js

```diff
@@ -1,7 +1,27 @@
 import React from "react";
 
 const About = () => {
-  return <div>About</div>;
+  return (
+    <div>
+      <header>About</header>
```

Comment:
```suggestion
      <h1>About</h1>
```
`About` should be a heading (`h1`) rather than a header. A header contains child elements (see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header for an example)